hilti/src/rt/tests/util.cc

```diff
@@ -361,9 +362,11 @@ TEST_CASE("map_tuple") {
     CHECK_EQ(input, std::make_tuple(2u, 2L, std::string("aa")));
 }
 
-TEST_CASE("memory_statistics") {
-    auto ms = memory_statistics();
+TEST_CASE("resource_usage") {
+    auto ms = resource_usage();
 
+    usleep(10000);
```

Comment:
Why is this needed? I'd imagine this could be useful if we'd test for a change in resource usage, but we don't.

Reply:
I saw test failures for the new `> 0` check on execution time so far, presumably because the double couldn't represent whatever tiny number of execution had passed at that point.